manifest/2.7.xml

```diff
@@ -13,15 +13,15 @@
 
     <!-- Build Scripts (required on CI servers) -->
     <project name="product-texts" path="product-texts" remote="couchbase"/>
-    <project name="build" path="cbbuild" remote="couchbase">
-        <annotation name="VERSION" value="2.7.1"     keep="true"/>
+    <project name="build" path="cbbuild" remote="couchbase" revision="a08bf70a05fc5b94e62c6aa2d349d3f1e261f1cc">
```

Comment:
Nice job catching that the release build (2.7.1-5) had a different build SHA than the latest on build-manifests/sync_gateway.

Should the build version in the 2.7.1.xml manifest be updated to the same thing?

Reply:
Heh, good catch. Updated.